lib/cache/cache_test.go

```diff
@@ -1268,8 +1275,6 @@ func TestClusterName(t *testing.T) {
 
 // TestNamespaces tests caching of namespaces
 func TestNamespaces(t *testing.T) {
-	t.Parallel()
```

Comment:
Why is this removed? What is changed that this test can't be run in parallel anymore?

Reply:
Ahhh whoops, that wasn't intentional! I'll re-add that.